src/tests/groupby/groupby-kernels-test.cu

```diff
@@ -392,38 +392,40 @@ TYPED_TEST(GroupByTest, DISABLED_AggregationTestHost)
   thrust::pair<key_type, value_type> second_pair{0,10};
   thrust::pair<key_type, value_type> third_pair{0,5};
 
-  auto max = [](value_type a, value_type b) { return (a >= b ? a : b); };
+  struct {
+    __device__ value_type operator()(value_type a, value_type b) { return (a >= b ? a : b); };
+  } maxop;
 
-  this->the_map->insert(first_pair, max);
+  this->the_map->insert(first_pair, maxop);
```

Comment:
I believe this won't work. The `concurrent_unordered_map::insert` function can be called either from the device or the host, the implementation is determined by an `#ifdef __CUDA_ARCH__`. 

In this test, the function is being called from the host, but with a `__device__` functor.

This is irrelevant moving forward as I will be removing the host implementation of the `concurrent_unordered_map::insert` function because it is not needed and requires the class to be allocated with managed memory. As a result, I will be removing this test as well.

Reply:
Oops, I missed the __host__ specifier on this. Funny that the tests passed anyway, Jake! Anyway, I have fixed it with latest commit. Crap, just noticed this is already merged! Maybe you can add the __host__ to the functor next time you work on this file (or remove the test).